### PR TITLE
Jar with dependencies

### PR DESCRIPTION
**Pull request by SeriyBg**
Change maven build so that the jar includes hazelcast dependency

### CODE REVIEW

File: src/main/java/com/hazelcast/jdbc/Driver.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public Logger getParentLogger() throws SQLFeatureNotSupportedException {
         throw new SQLFeatureNotSupportedException("The driver does not use java.util.logging");
     }
 
-    private static synchronized void load() {
```
Comment (devozerov) [inline src/main/java/com/hazelcast/jdbc/Driver.java:91]: Why do we remove the synchronization here?

Comment (SeriyBg) [reply src/main/java/com/hazelcast/jdbc/Driver.java:91]: It's removed according to #46

File: META-INF/maven/com.hazelcast/hazelcast-jdbc/pom.xml [hunk 2]
```diff
@@ -0,0 +1,442 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.hazelcast</groupId>
+    <artifactId>hazelcast-jdbc</artifactId>
+    <version>1.0-SNAPSHOT</version>
```
Comment (viliam-durina) [inline META-INF/maven/com.hazelcast/hazelcast-jdbc/pom.xml:9]: Use 4.2-SNAPSHOT

**Review by viliam-durina — APPROVED** [review]
I don't understand much of the maven lingo, but built the artifact and tried to connect to both IMDG and Jet cluster and it works..

File: META-INF/maven/com.hazelcast/hazelcast-jdbc/pom.xml (hunk 2 above)
Comment (SeriyBg) [reply META-INF/maven/com.hazelcast/hazelcast-jdbc/pom.xml:9]: Done here #57

**Review by devozerov — APPROVED** [review]

**Pull request merged**